test/datastream/datastream_test.go

```diff
@@ -75,6 +75,11 @@ var _ = Describe("Streaming Data Conversion", func() {
 			desc := t.testDesc
 			ff := t.expectFormats
 			fn := t.inFileName
+			fnbase := fn
```

Comment:
This isn't really the base filename. I'd use `filepath.Ext` to get the extension string then extract the remainder, eg:
```
ext := filepath.Ext(fn)
fnbase  := f[0:len(f)-len(ext)]
```